docs/environments/butterfly.md

```diff
@@ -50,7 +50,7 @@ env.close()
 ```
 
 To launch a [Knights Archers Zombies](/environments/butterfly/knights_archers_zombies/) environment with interactive user input (see [manual_policy.py](https://github.com/Farama-Foundation/PettingZoo/blob/master/pettingzoo/butterfly/knights_archers_zombies/manual_policy.py)):
-```python
+```python notest
```

Comment:
Make sure to add a todo for this or reason for not being tested

Reply:
I forgot to add it here but I get the error `ValueError: when an agent is dead, the only valid action is None` so its prob related to https://github.com/Farama-Foundation/PettingZoo/pull/1075#discussion_r1305750601

Reply:
Actually now that I look at it, it doesn't check for termination for each agent here. Adding the check from https://pettingzoo.farama.org/environments/butterfly/knights_archers_zombies/#aec removes the error.